sympy/printing/tests/test_latex.py

```diff
@@ -524,7 +524,7 @@ def test_latex_indexed():
     indexed_latex = latex(Psi_indexed[0] * conjugate(Psi_indexed[0]))
     # \\overline{{\\Psi}_{0}} {\\Psi}_{0}   vs.   \\Psi_{0} \\overline{\\Psi_{0}}
     assert symbol_latex=='\\Psi_{0} \\overline{\\Psi_{0}}' \
-        or indexed_latex=='\\overline{{\\Psi}_{0}} {\\Psi}_{0}'
+        and indexed_latex=='\\overline{{\\Psi}_{0}} {\\Psi}_{0}'
```

Comment:
This should not be a continuation line with `and` but `assert indexed_latex == ...`. The results are now different and each one should have its own test line.